exercises/D-logical-operators/exercise2.js

```diff
@@ -5,7 +5,37 @@
   Update the code so that you get the expected result.
 */
 
-function isNegative() {}
+function isNegative(number) {
+  if(number < 0){
+    return true;
+  } else {
+    return false;
+  }
+}
+
```

Comment:
I've already learned a very brief code for this function. I want to share it with you.

function isNegative(num) {
  return num <0;
  }

Reply:
Yes, that is much better.